third_party/freetype2/BUILD.gn

```diff
@@ -12,9 +12,16 @@ declare_args() {
 import("../third_party.gni")
 
 if (skia_use_system_freetype2) {
-  system("freetype2") {
-    include_dirs = [ "/usr/include/freetype2" ]
-    libs = [ "freetype" ]
+  if (is_tizen) {
+    system("freetype2") {
+      cflags = [ "-I=/usr/include/freetype2" ]
```

Comment:
Just checking to see if there was a reason for this specific change for Tizen?

Reply:
It was mistakenly left over. Fixed. I missed it. Thanks.